modules/sdk-coin-avaxp/src/delegatorTxBuilder.ts

```diff
@@ -0,0 +1,233 @@
+import { BaseAddress, BaseKey, BuildTransactionError, NotImplementedError, TransactionType } from '@bitgo/sdk-core';
+import { AvalancheNetwork, BaseCoin as CoinConfig } from '@bitgo/statics';
+import { Transaction } from './transaction';
+import { TransactionBuilder } from './transactionBuilder';
+import {
+  AddDelegatorTx,
+  BaseTx,
+  ParseableOutput,
+  SECPOwnerOutput,
+  SECPTransferInput,
+  SECPTransferOutput,
+  TransferableInput,
+  TransferableOutput,
+} from 'avalanche/dist/apis/platformvm';
```

Comment:
Is it possible to import without using `dist`?

Reply:
this was the error we were getting before when it wasn't importing correctly and we fixed it by using dist.  Any suggestions to get around it -- this assumes that you have to do a yarn install before running any of the packages in the module.

Reply:
When I try to change, I find problems with type checks. So we choose to avoid fix for now and keep it to future improve.
[STLX-17199](https://bitgoinc.atlassian.net/browse/STLX-17199) [sdk-coin-avaxp] Changes avalanche import path from dist for src

Reply:
We will not be able to import without using dist.  Just messaged avalanche foundation and importing from avalanche/dist has all the transpiled JS.

Reply:
Yeah, we don't want to import from `src` which we already changed. I'm generally concerned that we even have to import from `dist` from the avalanche libraries since it should be automatic from the root of the package in node_modules. Using `dist/filename` is a work around for accessing things that should have already been available from the root.

Reply:
main issue is that typeID is defined as a number then defined as defined as [unknown](https://github.com/ava-labs/avalanchejs/blob/master/src/utils/serialization.ts#L49-L51).  When using typescript, and importing from src this part breaks.  Avalanche foundation says that not all transactions have a typeID so they left it undefined.  For all of our transactions we require this typeID